plutus-tx-plugin/src/PlutusTx/Compiler/Laziness.hs

```diff
@@ -42,7 +38,10 @@ delayVar (PIR.VarDecl () n ty) = do
 force
     :: CompilingDefault uni fun m
     => PIRTerm uni fun -> m (PIRTerm uni fun)
-force thunk = PIR.Apply () thunk <$> compileExpr (GHC.Var GHC.unitDataConId)
+force thunk = do
+    a <- liftQuote (freshTyName "dead")
```

Comment:
It seems a bit morbid having all this death everywhere. I supposed it vanishes during erasure though, so at least it won't show up in the final output.